app/controllers/admin/custom_organisations_controller.rb

```diff
@@ -12,6 +14,7 @@ def create
 
     if @custom_organisation.save
       flash.now[:notice] = "Custom organisation has been successfully added"
+      set_organisations_from_register
```

Comment:
Isn't this run in the before action already?

Reply:
Hahah let me explain

Reply:
Easier face to face

Reply:
As its bit odd